packages/markdown-docx/src/ToOOXMLVisitor/index.js

```diff
@@ -59,6 +63,46 @@ class ToOOXMLVisitor {
         return node.$class;
     }
 
+    /**
+     * Generates the OOXML for text and code ciceromark nodes.
+     *
+     * @param {string} value           Text value of the node
+     * @param {Array}  nodeProperties  Properties of the node
+     * @param {string} defaultProperty Default property of the node
+     * @returns {striing} Generated OOXML
+     */
+    generateTextOrCodeOOXML(value, nodeProperties, defaultProperty = '') {
+        let propertyTag = defaultProperty;
```

Comment:
What is the point of this? You can directly use the argument's name.

Reply:
@algomaster99 
Didn't understand what you mean.

Reply:
I am questioning the assignment of defaultProperty argument to `propertyTag`. You can just use defaultProperty instead of propertyTag.